### PR TITLE
fix: output otel ruby sdk version

### PR DESCRIPTION
**Pull request by xuan-cao-swi**
## Description

Output otel ruby sdk version for component-version

### CODE REVIEW

File: .github/workflows/release-layer-ruby.yml [hunk 1]
```diff
@@ -19,12 +19,20 @@ jobs:
...
         working-directory: ruby/src
 
+      - name: Save Ruby SDK Version
+        id: save-ruby-sdk-version
+        run: |
+          unzip -q build/opentelemetry-ruby-layer.zip
+          export GEM_PATH=$PWD/ruby/gems/3.2.0/
+          RUBY_SDK_VERSION=$(ruby -e 'require "opentelemetry-sdk"; puts OpenTelemetry::SDK::VERSION')
+          echo "RUBY_SDK_VERSION=$RUBY_SDK_VERSION" >> $GITHUB_OUTPUT
+        working-directory: ruby/src
```
Comment (tylerbenson) [inline .github/workflows/release-layer-ruby.yml:34]: Mind testing it on a personal fork?  You can temporarily disable the call to the publish workflow.

Comment (xuan-cao-swi) [reply .github/workflows/release-layer-ruby.yml:34]: Sure, although I already did similar test on my [test repo](https://github.com/xuan-cao-swi/test-action-release/actions/runs/10009575266), but I am happy to do it again on my personal fork.

Do I just comment out the `publish_layer` job and add `workflow_dispatch:` for testing, and revert back once it's ready?

Comment (tylerbenson) [reply .github/workflows/release-layer-ruby.yml:34]: That's good enough for me.

**Review by tylerbenson — APPROVED** [review]

**Comment by tylerbenson** [comment]
@xuan-cao-swi looks like these releases are still labeled as `dev`, so I've gone back and marked them as pre-releases.

Can you please confirm the releases are working as expected and remove `release-group: dev`?

Thanks!

**Pull request merged**